src/c_bindings/c_wrapper.cpp

```diff
@@ -22,7 +22,7 @@ extern "C" {
             std::vector<uint8_t> challenge_hash_bytes(challenge_hash, challenge_hash + challenge_size);
             integer discriminant = CreateDiscriminant(challenge_hash_bytes, discriminant_size_bits);
             form x = DeserializeForm(discriminant, x_s, x_s_size);
-            std::vector<uint8_t> result = ProveSlow(discriminant, x, num_iterations);
+            std::vector<uint8_t> result = ProveSlow(discriminant, x, num_iterations, "");
```

Comment:
ideally, `prove_wrapper()` would take a `char const*` and you would turn that into a `std::string` (where `nullptr` means an empty string). We're working towards using the rust binding, so we'd have to add this eventually anyway

Reply:
unfortunately i couldn't get the rust tests to work due to my lack of rust skills, thats why it is hacked to the original implementation here